doc/sources/installation/installation-linux-venvs.rst

```diff
@@ -101,8 +102,8 @@ After you ensure that a 3rd-party repository containing any packages that dnf is
 
     sudo pip3 install --upgrade pip setuptools
 
-    # Use correct Cython version here (0.28.2 is for 1.10.1):
-    sudo pip3 install Cython==0.28.2
+    # Use correct Cython version here (|cython_install| is for 1.10.1):
```

Comment:
thanks Matham for enhancing the installation docs!
... but I think the text here should read: "... (|cython_install| is for the actual kivy version)"

Reply:
Thanks for letting me know. It is fixed in https://github.com/kivy/kivy/commit/1918b975511396c701ad889740d51ab2fdf0490c.